test/cpp/interop/xds_interop_client.cc

```diff
@@ -51,6 +53,8 @@ DEFINE_int32(rpc_timeout_sec, 30, "Per RPC timeout seconds.");
 DEFINE_string(server, "localhost:50051", "Address of server.");
 DEFINE_int32(stats_port, 50052,
              "Port to expose peer distribution stats service.");
+DEFINE_int32(config_port, 50053,
```

Comment:
How does C-core's gRPC server work? Does it have to be one server for one service? In Java, one server can have multiple services registered. So we just registered both stats service and config service to the same server instance. The test driver will be using the same port for invoking both kinds of RPCs to test client. We need this to be consistent across languages. Otherwise, the test driver will not work.

Reply:
I agree, this should be changed to run both services on the same server.  There's no reason to require them to be on different ports.

Reply:
fixed, reusing the same port